charts/testkube-operator/templates/executor.testkube.io_executors.yaml

```diff
@@ -14,119 +15,127 @@ spec:
+                type: array
+              executor_type:
+                description: ExecutorType one of "rest" for rest openapi based executors
+                  or "job" which will be default runners for testkube or "container"
+                  for container executors
+                enum:
+                - job
+                - container
+                type: string
+              features:
+                description: Features list of possible features which executor handles
+                items:
+                  enum:
+                  - artifacts
+                  - junit-report
                   type: string
-                meta:
-                  description: Meta data about executor
+                type: array
+              image:
+                description: Image for kube-job
+                type: string
+              imagePullSecrets:
+                description: container executor default image pull secrets
+                items:
+                  description: LocalObjectReference contains enough information to
+                    let you locate the referenced object inside the same namespace.
                   properties:
-                    docsURI:
-                      description: URI for executor docs
-                      type: string
-                    iconURI:
-                      description: URI for executor icon
+                    name:
+                      description: 'Name of the referent. More info: https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#names
+                        TODO: Add other useful fields. apiVersion, kind, uid?'
                       type: string
-                    tooltips:
-                      additionalProperties:
-                        type: string
-                      description: executor tooltips
-                      type: object
                   type: object
-                types:
-                  description: Types defines what types can be handled by executor e.g.
-                    "postman/collection", ":curl/command" etc
-                  items:
+                type: array
+              job_template:
+                description: Job template to launch executor
+                type: string
+              jobTemplateReference:
+                description: name of the template resource
+                type: string
+              meta:
+                description: Meta data about executor
+                properties:
+                  docsURI:
+                    description: URI for executor docs
+                    type: string
+                  iconURI:
+                    description: URI for executor icon
+                    type: string
+                  tooltips:
+                    additionalProperties:
+                      type: string
+                    description: executor tooltips
+                    type: object
+                type: object
+              slaves:
```

Comment:
Is the following correct:

`slaves` object is optional, but if it is provided, then it must contain the `image` field?

Reply:
Yes